src/solve.jl

```diff
@@ -75,7 +75,7 @@ function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algTyp
                                                        ode_prob,
                                                        OrdinaryDiffEq.alg_order(alg)))
     end
-    integrator.dt = dt
+    integrator.dt = zero(dt)
```

Comment:
No, this just assures that the ODE integrator always satisfies `tprev + dt == t`. The DDE integrator is still initialized with the init dt choice from the user (or the calculated init dt).